src/main/java/streams/Main.java

```diff
@@ -0,0 +1,51 @@
+package streams;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+
+public class Main {
+    public static void main(String[] args) throws IOException {
+        String[] keywords = {"abstract", "assert", "boolean", "break", "byte", "case",
```

Comment:
Extract the `keywords` array to a constant or a separate utility class to improve readability and maintainability.

Reply:
Use Set\<String\> and think about why it's better.

Reply:
I extracted it into a constant, is there a difference in the solution if you make a service class?
Set guarantees the uniqueness of the elements, algorithmic complexity for the HashSet search operation O(1)